src/plone/api/content.py

```diff
@@ -666,3 +667,52 @@ def find(context=None, depth=None, unrestricted=False, **kwargs):
         return catalog.unrestrictedSearchResults(**query)
     else:
         return catalog(**query)
+
+
+@required_parameters("obj")
+def get_parents(obj: None, predicate: None, interface: None):
+    """ Get all parents of an object, with optional filtering. 
+    
+    :param obj: [required] Object for which we want to get the parents.
+    :type obj: Content object
+    :param predicate: Optional callable that takes an object and returns a boolean.
+        Used to filter the parents.
+    :type predicate: callable
+    :param interface: Optional interface to filter the parents.
+    :type interface: zope.interface.Interface
+    :returns: List of parent objects, from immediate to site root.
+    :rtype: list
+    :Example: :ref:`content-get-parents-example`
+    
+    """
+    chain= aq_chain(obj)[1:]
```

Comment:
Maybe this one should be:
```suggestion
    chain= aq_chain(aq_inner(obj))[1:]
```